src/include/network/lan/cmdline.rb

```diff
@@ -172,7 +178,7 @@ def DeleteHandler(options)
     # @param options [Hash{String => String}] action options
     # @return [String] infered device type; an empty string if not infered
     def infered_type(options)
-      return "bond" if options.include? "slaves"
+      return "bond" if options.include? "bond_ports"
```

Comment:
```suggestion
      return "bond" if options.include? "slaves"
      return "bond" if options.include? "bond_ports"
```

Reply:
Hopefully not. That's reason I've added https://github.com/yast/yast-network/pull/1280/commits/7a5a3278932d6ce097aaaf8cfa7b5e9fb55c6fd9#diff-cd51e63dfa3bcb1671b7efb41b3f76ead7339528b230c594ae3882029620ee75R112

Reply:
Yes, I see now.

May I point out that my way is less code and easier to understand and review? :-)

Reply:
yep. But my way makes behavior cleaner when both `slaves` and `bond_ports` are present simultaneously